awsDetonationLab.template

```diff
@@ -876,107 +876,27 @@
       }
     },
     "FlowLogUpload": {
-      "Type": "AWS::Lambda::Function",
+      "Type": "AWS::EC2::FlowLog",
       "Properties": {
-        "Handler": "index.handler",
-        "MemorySize": 256,
-        "Role": {
+        "DeliverLogsPermissionArn": {
```

Comment:
According to the [AWS::EC2::FlowLog documentation](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-flowlog.html#w2ab1c21c10d102c40b9):
> If you specify `LogDestinationType` as `s3`, do not specify `DeliverLogsPermissionArn` or `LogGroupName`. 

Which means this field is no longer necessary and must be removed.

Reply:
removed vpc lambda related resources from template

Reply:
Aren't those vpc lambda related too?
https://github.com/sonofagl1tch/AWSDetonationLab/blob/a681a7465d78f038f6aba1ee54a4cdcbfe185d10/awsDetonationLab.template#L847-L851
https://github.com/sonofagl1tch/AWSDetonationLab/blob/a681a7465d78f038f6aba1ee54a4cdcbfe185d10/awsDetonationLab.template#L879-L917
https://github.com/sonofagl1tch/AWSDetonationLab/blob/a681a7465d78f038f6aba1ee54a4cdcbfe185d10/awsDetonationLab.template#L867-L878